.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -41,23 +89,49 @@ If you need to submit a pr for any of these repos during this period, you should
   </ul>
 </details>
 
-- [ ] **Enable code freeze bot** In order to enable the code freeze GitHub you need to change the line `if: github.repository_owner == 'adoptium' && false` to be `if: github.repository_owner == 'adoptium' && true` in the [code-freeze.yml](https://github.com/adoptium/.github/blob/main/.github/workflows/code-freeze.yml#L21) GitHub workflow. Please contact the PMC if you need help merging this change.
-- [ ] **Disable nightly build completely** by disabling the pipelines temporarily to free up resources and ensure no competing jobs during release week
-- [ ] Tag the build repositories (temurin-build, ci-jenkins-pipelines, jenkins-helper) for the release level tooling: github/\<repo\>/releases, "Draft new release". Choose tag: "v\<YYYY\>.\<MM\>.01" (increment 01 if needed..)
+ - [ ] **Enable code freeze bot** In order to enable the code freeze GitHub you need to change the line `if: github.repository_owner == 'adoptium' && false` to be `if: github.repository_owner == 'adoptium' && true` in the [code-freeze.yml](https://github.com/adoptium/.github/blob/main/.github/workflows/code-freeze.yml#L21) GitHub workflow. Please contact the PMC if you need help merging this change.
+ - [ ] **Freeze The Non Tagged Repositories** implement code freeze on the following repositories : 		
+	 - github-release-scripts https://github.com/adoptium/github-release-scripts
+	 - containers https://github.com/adoptium/containers
+	 - installer https://github.com/adoptium/installer
+	 - mirror-scripts https://github.com/adoptium/mirror-scripts
+
+- [ ] **Disable nightly builds completely** by disabling the pipelines temporarily to free up resources and ensure no competing jobs during release week
```

Comment:
maybe mention it is standard nightly and the evaluation nightly both

Reply:
As new reader, question comes to mind: How do I disable the pipelines? 

Also might be worth explicitly listing pipeline names all together (for nightly & weekly). E.g replace disable nightly/weekly with just:

Disable the following pipelines:
...